ecl/hqlcpp/hqlhtcpp.ipp

```diff
@@ -299,4 +299,14 @@ IHqlExpression * extractFilterConditions(HqlExprAttr & invariant, IHqlExpression
 bool isLibraryScope(IHqlExpression * expr);
 extern IHqlExpression * constantMemberMarkerExpr;
 
+inline void addFileAccessSignedAttribute(ActivityInstance * instance, IHqlExpression * signedAttr)
```

Comment:
minor: Cleaner as a member function of ActivityInstance, and no need to be inline.